pkgs/by-name/st/stash-clipboard/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
```

Comment:
`buildRustPackage` supports the `finalAttrs` pattern, which I think is prefferaable for this context. You can also put `version` in a `let  in` block and avoid both altogether.